service-front/assets/sass/utilities/_helpers.scss

```diff
@@ -86,6 +86,9 @@
         display: none !important;
         visibility: hidden !important;
     }
+
+    display: unset !important;
```

Comment:
Not sure we need this.

Reply:
If we want the no js message appearing/disappearing correctly we need these values in the CSS